### ISSUE TITLE
Can't install on Debian Jessie

### ISSUE THREAD

**Issue by nalipaz** [danfruehauf/NetworkManager-ssh#55]
Trying to install I get the following issues. I am using the latest release 1.2.0, let me know if it is better to use latest HEAD:

```
$ sudo apt-get install libnm-glib-dev libnm-glib-vpn-dev libnm-util-dev libnm-dev libnma-dev libgnome-keyring-dev dh-autoreconf libgtk-3-dev sshpass
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package libnm-dev
E: Unable to locate package libnma-dev
```
I then attempt to correct with:
```
$ sudo apt-get install libnm-glib-dev libnm-glib-vpn-dev libnm-util-dev libgnome-keyring-dev dh-autoreconf libgtk-3-dev sshpass libnm-gtk-dev libsecret-1-dev
```
That goes fine, so I proceed with the compile:
```
$ autoreconf -fvi && ./configure && make deb
```
...
checking for uname... yes
./configure: line 12107: syntax error near unexpected token `0.35'
./configure: line 12107: `IT_PROG_INTLTOOL(0.35)'
```
Error.

Not sure how to proceed. Thanks.

**Comment by danfruehauf** [comment]
I think best to use upstream packages for that. I should also update the README. Also, best to report bugs against the distro you're using.

@lhw Can you add more information please or point @nalipaz in the right direction? :+1:

**Comment by lhw** [comment]
@nalipaz
That command will not work on stable (jessie). The 1.2.0 release requires libraries from
testing/unstable. It won't build with the packages in stable.
I'm currently waiting on the upload of the 1.2.0-1 package I created from the tagged release.
I'd say wait a few days and then install it from unstable or download the new package
right now with this command (still requires testing/unstable network-manager packages):

dget -x
https://mentors.debian.net/debian/pool/main/n/network-manager-ssh/network-manager-ssh_1.2.0-1.dsc

At least it will also pull in the dependencies you need. Otherwise install the 0.9.4 release from
testing right now. That should work with the stable packages.

@danfruehauf It's not really a distro bug as he is trying to manually install the
build-dependencies and using the makefile to build the deb. He is just ending up with outdated
packages which won't allow him to build the plugin in the end.

**Comment by nalipaz** [comment]
@lhw generally mixing stable and upstream is discouraged, so I attempted to download the 0.9.4 binary from packages.debian.org. Unfortunately, installing with gdebi couldn't resolve dependencies it needed. libdbus-1-3 and libdbus-1-dev. I ended up stopping at that point. 

Well, I hope to see it become available in jessie-backports or perhaps a method of compiling that might work under jessie.

**Comment by danfruehauf** [comment]
> It's not really a distro bug as he is trying to manually install the build-dependencies and using the makefile to build the deb. He is just ending up with outdated packages which won't allow him to build the plugin in the end.

@lhw It's actually likely that I'll remove the "debian/ubuntu support" from this repository, as it didn't update for a while and things have probably changed. Since there are also distro maintainers for it now (you :wink:) it is unnecessary.

**Comment by danfruehauf** [comment]
> Well, I hope to see it become available in jessie-backports or perhaps a method of compiling that might work under jessie.

I should look into that. Generally I'm happy to receive patches for that and push them to the correct branch. Unfortunately I'm not a debian user so it makes debugging on my end not that easy...

**Comment by lhw** [comment]
@nalipaz I'm well aware of the franken debian problem. But it will take some time before somebody backports it to stable and I don't see the harm if you maintain a good apt preferences file.

**Comment by nalipaz** [comment]
Well, I would rather not mix things from stable and testing. Mainly because of the dependencies that would also need to be pulled in from testing, I can't be sure how those might affect other applications in the OS. 

I guess for now I will be content with using the terminal to make my ssh tunnel connection and then using foxyproxy to route to it. 

I do think that the debian instructions should be removed since currently there is no way it can safely work in stable.

**Comment by nalipaz** [comment]
By the way, if I used an older release would it work on Jessie?

**Comment by danfruehauf** [comment]
> I do think that the debian instructions should be removed since currently there is no way it can safely work in stable.

I agree. I'll work on that as soon as I have a decent internet connection.

> By the way, if I used an older release would it work on Jessie?

Not sure, I'm very unfamiliar with Debian...

**Comment by danfruehauf** [comment]
Lets get a status update on that? My impression is that things work on debian/ubuntu, thanks to @lhw :pray:
